grouper/constants.py

```diff
@@ -24,6 +25,7 @@
     (PERMISSION_CREATE, "Ability to create permissions within Grouper."),
     (PERMISSION_GRANT, "Ability to grant a permission to a group."),
     (PERMISSION_AUDITOR, "Ability to own or manage groups with audited permissions."),
+    (AUDIT_SECURITY, "Ability to audit security related activity on the system."),
```

Comment:
Should be `security-related`. And is this going to be just for SSH keys, or will the scope be expanded later on?